handbook/ce/engaging-other-teams.md

```diff
@@ -1,5 +1,168 @@
+
+Questions about specific alerts and graph panels should be routed to the team that owns the alert or panel, as indicated by relevant entry in [Alert solutions](https://docs.sourcegraph.com/admin/observability/alert_solutions) or the [Dashboards reference](https://docs.sourcegraph.com/admin/observability/dashboards) respectively.
+
+Any other questions about monitoring and performance should be routed to the [Distribution team](../engineering/distribution/index.md). 
+
+
+### Code host connections
+
+**Keywords**: `code host`, `cloning`, `syncing`, `token`, `GitHub`, `GitLab`, `Bitbucket`, `Phabricator`, `Gerrit`, `repository`, `project`, `Perforce`, `src-expose`
+
+Any questions about code host connections and repository syncing should be routed to the [Core application team](../engineering/core-application/index.md).
+
+Note that this section applies to backend connections with code hosts, such as repository cloning and syncing. Questions about [frontend/UI integrations with code hosts](#browser-extensions-and-code-host-native-integrations) (e.g., getting code intelligence inside of a code host) should be rounted to the [Web team](../engineering/web/index.md).
+
+### Repository permissions
+
+**Keywords**: `permissions`, `ACLs`, `access`, `authorization`, `authz`
+
+Any questions about repository permissions should be routed to the [Core application team](../engineering/core-application/index.md).
+
+### User authentication (SSO)
+
+**Keywords**: `authentication`, `authn`, `SSO`, `SAML`, `OAuth`, `auth proxy`, `OpenIDConnect`, `OIDC`
+
+Any questions about user authentication should be routed to the [Core application team](../engineering/core-application/index.md).
+
+### Code intelligence
+
+**Keywords**: `code intelligence`, `precise`, `navigation`, `LSIF`, `language server`/`LSP`, `go to definition`, `hover`, `find references`, `ctags`, any programming language names (e.g. `Go`, `Java`, `Python`, `COBOL`, etc.)
+
+Any questions about code intelligence and navigation should be routed to the [Code Intelligence team](../engineering/code-intelligence/index.md).
+
+### Search
+
+**Keywords**: `search`, `indexed search`, `indexing`, `diff search`, `symbols`, `keyword`, `filter`, `scope`, `version context`, `repogroup`, `saved search`, `code monitoring`
+
+Any questions about search should be routed to the [Search team](../engineering/search/index.md).
+
+### Browser extension and code host native integrations
+
+**Keywords**: `browser extension`, `native integration`, `chrome`, `firefox`, `safari`, `Phabricator`, `Bitbucket`, `GitHub`, `GitLab`
+
+Any questions about the browser extension or code host native integrations should be routed to the [Frontend Platform team](../engineering/web/index.md).
+
+### Sourcegraph extensions
+
+**Keywords**: `extensions`, `plugins`, `blame`, `git-extras`, `Codecov`
+
+Any questions about Sourcegraph extensions should be routed to the [Extensibility team](../engineering/web/index.md).
+
+### Campaigns
```

Comment:
Just a heads-up: the name "campaigns" is almost certainly changing soon, and we will update this page when that happens. We will still want "campaigns" in the keywords, though.

Q: Since campaigns was originally called "automation", do we want that as a keyword also? (I can't think of the last time I saw a customer use "automation", so maybe it's not necessary.)

Reply:
Good points, thanks, Chris! I have a reminder set for my team and I to do a monthly audit on our docs. So, if when the name change is official (saw the post about the change and making sure there are not negative connotations with "batch changes."), we forget to update this page, we will catch it soon enough. For now, I will add both "automation" and "batch changes" as key words (well, if batch changes isn't there -- about to check after I finish this reply).